help/en/package/jmri/jmrit/withrottle/Protocol.shtml

```diff
@@ -632,11 +632,10 @@ MT+S3&lt;;&gt;ED&amp;RGW 341
       either the address directly ("S79" or "L3002") or by referencing
       a roster entry using the "E" command (such as "ED&amp;RGW 341").</p>
 
-      <p>Note that while this key must be formatted like an address, it
-      is <b>not</b> directly related to the actual address in use.  For example,
-      sending the command "MT+S23<;>L3000" will attempt to select the
-      locomotive with long address 3000, but will refer to it in
-      subsequent communication as "S23".</p>
+      <p>The key address MUST match the address chosen via the
```

Comment:
@bobjacobsen The server sends the roster list, which includes both the roster entry name "D&RGW 341" and the corresponding DCC address for that roster entry "S3". EngineDriver keeps that relationship, and sends back both when a roster entry is requested. Multiple roster entries can have the same address. If an address is requested without a roster entry name, EngineDriver places the address in both "parms".

Reply:
If an address is selected without a roster entry and is using the address directly, an application must make the two instances match in the connection line (MT+S23<;>S23), because the two servers I have use different portions of the command string as the source of the address to select.   That's why I call a mismatched undefined behavior.   

I did use the same example as earlier in the file where the roster list was presented, to help someone who's reading the whole document.